robyn/argument_parser.py

```diff
@@ -35,7 +35,7 @@ def __init__(self) -> None:
             help="Set the log level name",
         )
 
-        self.args = self.parser.parse_args()
+        self.args, unknown = self.parser.parse_known_args()
```

Comment:
nice!! I wasn't aware of this function 👍

Reply:
I had some issues while debugging because some unknown flags were passed to the file, this method fixed the issue because it lets you ignore unknown flags :wink: